pyvista/utilities/helpers.py

```diff
@@ -14,10 +15,14 @@
 import pyvista
 from .fileio import from_meshio
 
-POINT_DATA_FIELD = 0
-CELL_DATA_FIELD = 1
-FIELD_DATA_FIELD = 2
-ROW_DATA_FIELD = 6
+
+class FieldAssociation(enum.Enum):
```

Comment:
I chose FieldAssociation because that's what vtk refers to it as. DataAssociation may be more clear for that case, I can change it tomorrow.

Reply:
Ah no, that's a good reason. If the vtk naming is not entirely insensible we shouldn't divert from it. I just checked reusing `vtk.vtkDataObject.FieldAssociations` but that's not a python enum, but a very limited enum generating class (https://vtk.org/pipermail/vtk-developers/2014-November/031017.html). What we could consider doing is below instead of using straight integers, use e.g. `POINT = vtk.vtkDataObject.FieldAssociations(0)`, but the benefit would probably be minimal to non-existent.

Reply:
The enum values do exist in vtk, just not as part of the enum itself, but global variables: https://discourse.vtk.org/t/wrap-enums-as-pythons-enum-enum/2909/8

This means below instead of `POINT = 0` you can use `POINT = vtk.vtkDataObject.FIELD_ASSOCIATION_POINTS`.

Reply:
Agreed.  This might change in future versions as well, so we should set this here.

Reply:
FIELD association would be FIELD_ASSOCIATION_NONE, should I change FieldAssociation.FIELD to NONE also? That would be more clear to me and fix the clarity issue above.

Reply:
Yes, I'd change:
```FIELD = vtk.vtkDataObject.FIELD_ASSOCIATION_NONE```
to
```NONE = vtk.vtkDataObject.FIELD_ASSOCIATION_NONE```

Reply:
Ok, done.